content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,462 @@
+5. The grey setting of the scene disappears by default, but can be optionally reactivated. This meets a need for simplification for users following misunderstandings of the difference between global exposure and scene grey, both in terms of the visual result (there is none) and in terms of the overall workflow (caution, danger). The "canonical" way to use _filmic_ is therefore to use the exposure module to pre-adjust the overall exposure (brightness) of the image, and then to use _filmic_ only to constrain the dynamic range of the image within the available range of the display medium. In this configuration, dynamic range compression is done around the middle grey (conventionally set to 18%), which is the only value not affected by the transformation, thus preserving the overall brightness. Then, feel free to go beyond these recommendations, but it will cost you one more step to finally convince yourself that this is not a good idea. In this configuration, _filmic_ no longer affects the overall brightness, but only the compression of extreme brightnesses.
+6. The default settings for _filmic_ are adjusted by the software according to the exposure bias stored in the photo metadata. Thus, for digital photographers who are accustomed to exposing to the right of the histogram, forcing a manual exposure compensation on the camera body to protect the highlights from clipping (which is the only good way to expose in digital photography, with all due respect to purists who don't understand the technological differences between photoelectric semiconductor and photo-sensitized silver halide ions) will have their pipeline, relative to the scene, set automatically for them (as a first approximation, because filmmakers still can't guess what the exact mid-grey value is for a given scene).
+
+_Filmic v3_ is still available via the last tab of _filmic rgb_. As well as it is possible to update an image from _filmic v3_ to v4 with the same tab. Here a preview of this new _filmic rgb_ and the selection of the version wanted via science color menu:
+
+  <div style="text-align: center">
+![filmic rgb v4]({filename}en/filmicv4.png)
+  </div>
+
+Note: the reconstruction of highlights is done by means of wavelet decomposition by iteration, which induces a certain performance penalty. It is possible to disable the reconstruction completely by setting a very high highlight threshold, as the program bypasses the reconstruction if no pixels are detected beyond this threshold. Nevertheless, given the difficulty of convincingly reconstructing highlights, this is an acceptable compromise.
```

Comment:
Worth mentioning (since this has probably changed since your first draft) that a high highlight threshold is now set by default to improve performance.

Reply:
I know. It's Aurelien part and he reviewed it after that change, like all negadoctor, workflows and filmic parts. So I let that as it is. And on what I understand, that's really +6 treshold that completely deactivate highlight reconstruction, so new defaut +3 let that part always acceptable (even if it is less than before).

Reply:
Perhaps a new sentence: "The current default setting mostly disables highlight reconstruction to improve performance"